ietf/meeting/tests_views.py

```diff
@@ -7645,6 +7652,20 @@ def test_proceedings(self):
         SessionPresentationFactory(document__type_id='recording',session=session)
         SessionPresentationFactory(document__type_id='recording',session=session,document__title="Audio recording for tests")
 
+        # Add various group sessions
+        groups = []
+        parent_groups = [
+                GroupFactory.create(type_id="area", acronym="gen"),
+                GroupFactory.create(acronym="iab"),
+                GroupFactory.create(acronym="irtf"),
+                ]
+        for parent in parent_groups:
+            groups.append(GroupFactory.create(parent=parent))
+        for acronym in ["rsab", "edu"]:
```

Comment:
why edu?

Reply:
To test the `training` section. But I just noticed that I haven't added `training` to the section test. I'll add that.

Reply:
edu is closed. The current instantiation is emodir

Reply:
Existing code filter for **training** as follows: `sessions.filter(group__acronym__in=['edu','iaoc'], type_id__in=['regular', 'other',])`.
Does that need an update?

Reply:
@rjsparks [emodir](https://datatracker.ietf.org/group/emodir/about/) is conlcluded as well?

Adding `emodir` to the test fails the current logic for the learning section, So that logic definitely needs to be updated.

Reply:
bah - the current is `eodir`